apps/test-suite/tests/NewNotifications.js

```diff
@@ -1261,6 +1261,49 @@ export async function test(t) {
         4000
       );
 
+      t.fit(
+        'schedules a repeating yearly notification; only first scheduled event is verified.',
+        async () => {
+          const dateNow = new Date();
+          const trigger = {
+            day: dateNow.getDate(),
+            month: dateNow.getMonth(),
+            hour: dateNow.getHours(),
+            minute: (dateNow.getMinutes() + 2) % 60,
+            repeats: true,
+          };
+          await Notifications.scheduleNotificationAsync({
+            identifier,
+            content: notification,
+            trigger,
+          });
+          const result = await Notifications.getAllScheduledNotificationsAsync();
+          delete trigger.repeats;
+          if (Platform.OS === 'android') {
+            t.expect(result[0].trigger).toEqual({
+              type: 'yearly',
+              channelId: null,
+              ...trigger,
+            });
+          } else if (Platform.OS === 'ios') {
+            t.expect(result[0].trigger).toEqual({
+              type: 'calendar',
+              class: 'UNCalendarNotificationTrigger',
+              repeats: true,
+              dateComponents: {
+                ...trigger,
+                timeZone: null,
+                isLeapMonth: false,
```

Comment:
will this break if someone tests this during a leap month?

Reply:
I'm not sure… in that case, the daily and weekly should also fail since they have the same property here. My guess is this shouldn't be a problem because we are not specifying the `year` in any of the cases, so there is no way iOS could know we are talking about a leap year